app/src/main/java/org/dharmaseed/android/PlaybackService.java

```diff
@@ -70,11 +81,27 @@ public void onCreate() {
                 .setHandleAudioBecomingNoisy(true)
                 .setUseLazyPreparation(true)
                 .setAudioAttributes(audioAttributes, true)
+                .setSeekBackIncrementMs(SEEK_AMOUNT_MS)
+                .setSeekForwardIncrementMs(SEEK_AMOUNT_MS)
+                .build();
+
+        // Create seek back/forward command buttons
+        CommandButton seekBackButton = new CommandButton.Builder()
+                .setDisplayName("Seek Back")
+                .setIconResId(R.drawable.ic_seek_back)
+                .setSessionCommand(new SessionCommand(COMMAND_SEEK_BACK, new Bundle()))
+                .build();
+
+        CommandButton seekForwardButton = new CommandButton.Builder()
+                .setDisplayName("Seek Forward")
+                .setIconResId(R.drawable.ic_seek_forward)
+                .setSessionCommand(new SessionCommand(COMMAND_SEEK_FORWARD, new Bundle()))
                 .build();
 
-        // Create a media session
+        // Create a media session with custom command buttons
         mediaSession = new MediaSession.Builder(this, mediaPlayer)
                 .setCallback(new SessionCallback())
+                .setCustomLayout(ImmutableList.of(seekBackButton, seekForwardButton))
```

Comment:
Lol I swear I tried this exact thing, or something very close to it, a few years back and could never get it to work! :man_shrugging: Thank you again for figuring it out!